Trellix/trellix-epo-on-prem/tests/epo_event_1048.json

```
@@ -0,0 +1,68 @@
+  },
+  "expected": {
+    "message": "<?xml version=\"1.0\"?>\n<EPOEvent>\n  <MachineInfo>\n    <AgentGUID>{7434b2ff-872c-42e3-88f1-3e09ae7e33ee}</AgentGUID>\n    <MachineName>DESKTOP01</MachineName>\n    <RawMACAddress>38596153343D</RawMACAddress>\n    <IPAddress>1.2.3.4</IPAddress>\n    <AgentVersion>5.8.2.929</AgentVersion>\n    <OSName>Windows 10</OSName>\n    <TimeZoneBias>-60</TimeZoneBias>\n    <UserName>John_Doe</UserName>\n  </MachineInfo>\n  <SoftwareInfo ProductName=\"ENDP_AM_1070LYNX\" ProductVersion=\"10.7.18.211\" ProductFamily=\"TVD\">\n    <CommonFields>\n      <Analyzer>ENDP_AM_1070LYNX</Analyzer>\n      <AnalyzerName>Trellix Endpoint Security</AnalyzerName>\n      <AnalyzerVersion>10.7.18.211</AnalyzerVersion>\n    </CommonFields>\n    <Event>\n      <EventID>1048</EventID>\n      <Severity>1</Severity>\n      <GMTTime>2024-12-20T15:35:05</GMTTime>\n      <LocalTime>2024-12-20T16:35:05</LocalTime>\n      <CommonFields>\n        <AnalyzerDATVersion>5922.0</AnalyzerDATVersion>\n        <AnalyzerDetectionMethod>OAS</AnalyzerDetectionMethod>\n        <AnalyzerEngineVersion>6720.10487</AnalyzerEngineVersion>\n        <DetectedUTC>2024-12-20T15:35:05Z</DetectedUTC>\n        <TargetFileName>/var/log/gitlab/sidekiq/current</TargetFileName>\n        <TargetProcessName>/opt/gitlab/embedded/bin/svlogd</TargetProcessName>\n        <TargetUserName/>\n        <ThreatActionTaken>IDS_THREAT_ACTION_ALLOW</ThreatActionTaken>\n        <ThreatCategory>ops</ThreatCategory>\n        <ThreatHandled>0</ThreatHandled>\n        <ThreatName/>\n        <ThreatSeverity>4</ThreatSeverity>\n        <ThreatType>IDS_ALERT_DET_TYP_NOT</ThreatType>\n      </CommonFields>\n    </Event>\n  </SoftwareInfo>\n</EPOEvent>",
+    "event": {
+      "action": "IDS_THREAT_ACTION_ALLOW",
+      "category": [
+        "intrusion_detection"
+      ],
+      "severity": 1,
+      "type": [
+        "info"
+      ]
```

[Improvement]
According to the [ECS specification](https://www.elastic.co/guide/en/ecs/8.17/ecs-allowed-values-event-category.html#ecs-event-category-intrusion_detection), we can use the value `allowed` here (if we refer to the action in the event):
```suggestion
      "type": [
        "allowed"
      ]
```